### PR TITLE
operators [N] [CI] hazelcast-platform-operator (5.0.0)

### PR DESCRIPTION
**Pull request by devOpsHelm**

### CODE REVIEW

File: operators/hazelcast-platform-operator/5.0.0/metadata/annotations.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  operators.operatorframework.io.bundle.metadata.v1: metadata/
+  operators.operatorframework.io.bundle.package.v1: hazelcast-platform-operator
+  operators.operatorframework.io.bundle.channels.v1: alpha
+  operators.operatorframework.io.metrics.builder: operator-sdk-v1.13.0+git
+  operators.operatorframework.io.metrics.mediatype.v1: metrics+v1
+  operators.operatorframework.io.metrics.project_layout: go.kubebuilder.io/v3
+
+  # Annotations for testing.
+  operators.operatorframework.io.test.mediatype.v1: scorecard+v1
```
Comment (mvalarh) [inline operators/hazelcast-platform-operator/5.0.0/metadata/annotations.yaml:13]: @devOpsHelm please remove this 2 lines if you don't have tests

Comment (devOpsHelm) [reply operators/hazelcast-platform-operator/5.0.0/metadata/annotations.yaml]: Hi @mvalarh, I changed it as you suggested but the following test fails `TASK [deploy_olm_operator_upstream_bundle : Wait for the operator hazelcast-platform-controller-manager pod to start up]`. Any idea how can I get more information?

**Comment by devOpsHelm** [comment]
Hi @J0zi and @mvalarh,

I could not get the tests running on my local and I didn't get the problem by looking at the PR test logs. Could you help me with identifying the problem?

**Comment by devOpsHelm** [comment]
Hi @J0zi and @mvalarh,

When I try running the tests on my local the tests get stuck at `[kiwi] Reseting kind cluster ...`, I could not find a solution to this. Can you help me with finding the problem with the PR tests?

**Comment by J0zi** [comment]
@devOpsHelm , `repository` is not defined, check [preview](https://operatorhub.io/preview) and respected panel on the right.

Issue seems to be [CSV version parsing error](https://github.com/k8s-operatorhub/community-operators/runs/4618149877?check_suite_focus=true#step:6:1488)

**Comment by devOpsHelm** [comment]
@J0zi, are you sure that is why the tests are failing? I followed the documentation [here](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-required-fields.md#required-fields-for-operatorhub) and it says that field is optional. If that is the case, I will surely add it.

**Comment by devOpsHelm** [comment]
@J0zi, I applied the fix you mentioned but the `kiwi` test failed again. I am using `operator-sdk` to build the bundle and I know the manifest files that I use to create the bundle are correct because I am using them to deploy the operator. Do you have an idea about why we are facing the problem?

**Comment by devOpsHelm** [comment]
@J0zi  and @mvalarh, 

I see that there are other new operators facing the same problem as I do, kiwi tests failing after 20 mins. Is this problem a known error? What can I do about it, could you help me?

**Comment by J0zi** [comment]
@devOpsHelm  `Repository` was just an aesthetic issue. You can [debug](https://github.com/redhat-openshift-ecosystem/community-operators-pipeline/blob/documentation/docs/operator-test-suite.md) installation issue locally on k8s.

**Comment by devOpsHelm** [comment]
@J0zi, 

When I run the test using the following command

```
bash <(curl -sL https://raw.githubusercontent.com/redhat-openshift-ecosystem/community-operators-pipeline/ci/latest/ci/scripts/opp.sh) \ 
all operators/hazelcast-platform-operator/5.0.0
```

tests get stuck at the following step, do you know where can I get help for it?

```
[kiwi] Reseting kind cluster ...
```

**Comment by hasancelik** [comment]
Hi @J0zi,

I tried to run same command on MacOS with this branch but it failed for me also:

```
$ bash <(curl -sL https://raw.githubusercontent.com/redhat-openshift-ecosystem/community-operators-pipeline/ci/latest/ci/scripts/opp.sh) all operators/hazelcast-platform-operator/5.0.0

Info: No labels defined
debug=0
readlink: illegal option -- f
usage: readlink [-n] [file ...]
usage: dirname string [...]
usage: dirname string [...]
usage: dirname string [...]
Error : Unknwn value for 'OPP_STREAM=' !!!
```
Do you have any idea?

**Comment by J0zi** [comment]
@devOpsHelm did you executed following commands before curl? [See](https://github.com/redhat-openshift-ecosystem/community-operators-pipeline/blob/documentation/docs/operator-test-suite.md
```
OPP_AUTO_PACKAGEMANIFEST_CLUSTER_VERSION_LABEL=1 OPP_PRODUCTION_TYPE=k8s
```

**Comment by hasancelik** [comment]
@J0zi I am still getting the same error:
```
$ env | grep OPP
OPP_PRODUCTION_TYPE=k8s
OPP_AUTO_PACKAGEMANIFEST_CLUSTER_VERSION_LABEL=1
[hasancelik@Hasans-MacBook community-operators]$ bash <(curl -sL https://raw.githubusercontent.com/redhat-openshift-ecosystem/community-operators-pipeline/ci/latest/ci/scripts/opp.sh) all operators/hazelcast-platform-operator/5.0.0
Info: No labels defined
debug=0
readlink: illegal option -- f
usage: readlink [-n] [file ...]
usage: dirname string [...]
usage: dirname string [...]
usage: dirname string [...]
Error : Unknwn value for 'OPP_STREAM=' !!!
```
We sent this PR almost one month ago but still, it is not merged yet. Could you help us to merge our PRs?
Also, I have found that some `new operator` labeled PRs are already merged despite the `kiwi` job failure. For example, https://github.com/k8s-operatorhub/community-operators/pull/523

**Comment by hasancelik** [comment]
Hi @J0zi and @mvalarh,

Our release process is blocked for a month due to this PR and [other one](https://github.com/redhat-openshift-ecosystem/community-operators-prod/pull/541). Could you help us with local debug? Or could you provide more detailed info about the existing error?

**Comment by devOpsHelm** [comment]
@J0zi, I ran the following test 
```
bash <(curl -sL https://raw.githubusercontent.com/redhat-openshift-ecosystem/community-operators-pipeline/ci/latest/ci/scripts/opp.sh) \
kiwi operators/hazelcast-platform-operator/5.0.0

```
and it passed with following recap
```
PLAY RECAP *********************************************************************
localhost                  : ok=353  changed=40   unreachable=0    failed=0    skipped=254  rescued=0    ignored=2   
```

However, it still fails in the CI also the total number of tasks run differ between local and CI, what do you suggest we do?

**Comment by hasancelik** [comment]
> @J0zi, I ran the following test
> 
> ```
> bash <(curl -sL https://raw.githubusercontent.com/redhat-openshift-ecosystem/community-operators-pipeline/ci/latest/ci/scripts/opp.sh) \
> kiwi operators/hazelcast-platform-operator/5.0.0
> ```
> 
> and it passed with following recap
> 
> ```
> PLAY RECAP *********************************************************************
> localhost                  : ok=353  changed=40   unreachable=0    failed=0    skipped=254  rescued=0    ignored=2   
> ```
> 
> However, it still fails in the CI and also the total number of tasks in kiwi tests differ between local and CI, what do you suggest we do?

@J0zi and @mvalarh, could you help us? ☝️

**Comment by hasancelik** [comment]
Hi @J0zi and @mvalarh,

Could you help us with the CI error? As we mentioned above, we could not reproduce it locally.

Thanks,

**Comment by mvalarh** [comment]
I think your operator is not started. https://github.com/k8s-operatorhub/community-operators/runs/4883229510?check_suite_focus=true#step:6:1266

**Comment by mvalarh** [comment]
Maybe also : https://github.com/k8s-operatorhub/community-operators/runs/4883229510?check_suite_focus=true#step:6:1818

**Comment by mvalarh** [comment]
For local test for `k8s` can you run following?

```
OPP_PRODUCTION_TYPE=k8s bash <(curl -sL https://raw.githubusercontent.com/redhat-openshift-ecosystem/community-operators-pipeline/ci/latest/ci/scripts/opp.sh) \
kiwi operators/hazelcast-platform-operator/5.0.0
```

**Comment by fjammes** [comment]
It seems the bug comes from CI script, I have the very same in the PR for `qserv-operator`: https://github.com/k8s-operatorhub/community-operators/runs/4969640196?check_suite_focus=true, and I can see it in other PRs.

**Pull request merged**